### PR TITLE
Refactor of keyboard handler to start working on GUI

### PR DESCRIPTION
**Pull request by Andrej123456789**
As of [41th](https://github.com/ringwormGO-organization/Pelly/commit/110b232d7ffa8e93715376ef2ab382c2c50457b0) commit entire keyboard handler is in assembly. But, design choices of Pelly are making GUI in C and using Assembly for calls to BIOS. This pull request attempts that. Now you may wonder why we went back to Assembly. We used Assembly since then it was only option to make cursor keys to work. But this pull request will try to get cursor keys working in C too!

**DO NOT MERGE THIS PULL REQUEST YET**
I'll "un-draft" this pull request when will be ready for merge since now everything works as planned (C as main loop and calling Assembly for BIOS calls) except cursor keys which are going in wrong direction. That is likely from wrong `dh` & `dl` registers which I'll fix in later commits. Positions needed fix anyway and now I am doing it 2 in 1

And to @StjepanBM1 one note, use `dw` for [assigning numbers](https://www.tutorialspoint.com/assembly_programming/assembly_variables.htm) instead of `db`. And now I am getting normal numbers, ASCII numbers in C

**IN SHORT**
- by design choices, main loop goes in C and Assembly is used for BIOS calls
- this pr is not ready to merge since cursor & cursor positioning needs fix
- use `dw` instead of `db` for defining numbers in assembly

### CODE REVIEW

**Comment by Andrej123456789** [comment]
Cursor is fully working again. I fixed it by comparing as integer instead of as character.
If you are reading this now, you can merge this pull request or wait I fix starting positions which I'll fix very soon

**Comment by Andrej123456789** [comment]
I would appreciate if you can wait for PART 2 since there I'll change positions, in first part i just pass starting positions from C to asm and also I deleted unused `cursor_row` and cursor_column` variables. PART 2 will come for half an hour, one hour

**Comment by Andrej123456789** [comment]
Feel free to merge now!

**Comment by Andrej123456789** [comment]
Changes in short:
- main loop is in C
- required BIOS calls are called from assembly
- init keyboard takes two arguments for starting cursor positions

**Comment by StjepanBM1** [comment]
Well there are some issues that only happen in very weird cases, but other than that: everything looks fine to me. Merged
